lnrpc/walletrpc/walletkit.proto

```diff
@@ -220,46 +220,54 @@ message BumpFeeRequest {
 message BumpFeeResponse {
 }
 
+message SharedKeyRequest {
+    bytes ephemeral_pubkey = 1;
+}
```

Comment:
Could it be interesting to supply a key deriviation path for the node key, similar to `SignMessage`? Maybe not, because nobody on the network knows anything other than our usual pubkey.

Reply:
I don't see any use case for this at the moment.